src/store/modules/search.js

```diff
@@ -0,0 +1,65 @@
+import { fetchUsers } from '@/api/users'
```

Comment:
You dont need this module, you can use users,questions,tags directly

Reply:
Implemented this suggestion. Removed the search module, and used users, questions and tags modules. However, I decided to make separate lists for search fetches, because I don't want to mutate the same list. 

For example, If I am on users page, and make a search, If I use the same list in the module, I will mutate it with the search results and therefore mutate the user's page content. If I click on a user, there is no issue because I leave the users page, but what if I just change my mind and delete the search query? The search box should hide and I should see all users on the user's page again. 

Hope this is not too confusing, and please advise if this is the right approach!
Thank you